inc/command/class-command.php

```diff
@@ -1054,7 +1081,270 @@ protected function merge_config( array $config, array $overrides ) : array {
 				break;
 			}
 		}
+
 		return $merged;
 	}
 
+	/**
+	 * Bootstraps Docs linter configuration files.
+	 *
+	 * @param InputInterface $input
+	 * @param OutputInterface $output
+	 *
+	 * @return int
+	 */
+	protected function bootstrap_lintdocs( InputInterface $input, OutputInterface $output ) : int {
+
+		// Copy configuration file for Markdownlint
+		$target_file = getcwd() . '/.markdownlint.jsonc';
```

Comment:
Is it worth putting all these in a .lintdocs directory in the root of the project or module, rather than the current working directory?

Reply:
It probably is, but I've tried to use the various tools' default locations for things, but I can look at iterating on that, as I will need to for some things.